frontend/components/panels/MobileClassPanel.js

```diff
@@ -156,19 +126,18 @@ class MobileClassPanel extends BaseClassPanel {
     const showMoreSections = this.getShowMoreButton();
 
     // Decide which chevron to use based on whether the panel is expanded or not.
-    let chevron;
-    if (this.state.showMoreThanTitle) {
-      chevron = chevronDown;
-    } else {
-      chevron = chevronRight;
-    }
+    const chevron = (this.state.showMoreThanTitle) ? chevronDown : chevronRight;
 
     return (
       <div>
         <div className={ `${css.container} ui segment` }>
           <div
             className={ css.header }
-            onClick={ this.toggleShowMoreThanTitle }
+            onClick={ () => {
+              this.setState({
+                showMoreThanTitle: !this.state.showMoreThanTitle,
```

Comment:
This changes the behavior of clicking on the title so that it just acts as a hide/show toggle.

Previously, we had it reset to the initial state on close. However, it feels wrong to do so, for the following reasons:
1. If the user wishes to close it out and will never see it again, it makes no sense to do extra work to reset the state
2. If the user only wishes to temporarily close it out, they're likely trying to compare something on a small screen. If they're trying to compare something, it would be better to keep track of the state the panel is in, so that the user doesn't need to reselect every button.
3. The current layout implies that the class data is simply hidden away for layout sake; the user expects data to be hidden and shown, rather than reset.

It also makes the code a lot cleaner, since we can properly get rid of `getInitialState()`.